compatibility/history-proxy/js/license-exceptions.json

```diff
@@ -0,0 +1,37 @@
+{
+  "exceptions": {
+    "atob@2.1.2": {
+      "reason": "It allows to choose between MIT and Apache-2.0"
+    },
+    "type-fest@0.6.0": {
+      "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+    },
+    "type-fest@0.8.1": {
+      "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+    },
+    "type-fest@0.11.0": {
+      "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+    },
+    "type-fest@0.13.1": {
+      "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+    },
+    "type-fest@0.20.2": {
+      "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+    },
+    "type-fest@0.21.3": {
+      "reason": "MIT and/or CC0-1.0 are compatible with Apache-2.0"
+    },
+    "xmldom@0.1.27": {
+      "reason": "It allows to choose between LGPL-2.0 or MIT"
+    },
+    "tslib@1.14.1": {
+      "reason": "0BSD is public-domain-equivalent license, so it should be compatible with Apache-2.0"
+    },
+    "argparse@2.0.1": {
+      "reason": "Python-2.0 is compatible with Apache-2.0"
+    },
+    "caniuse-lite@1.0.30001207": {
+      "reason": "It's licensed under the CC-BY-4.0 License, appropriately labelled in package.json and no code changes, what makes compatible with Apache-2.0"
```

Comment:
There is no mention of the authors in this dependency, there should be at least one comment in the code (above `require()` maybe) mentioning the authors.
The `CC-BY` license says that it is necessary to mention the authors of the library. If possible, also include a comment in package.json.

Giving due credit is a request of those developers: https://github.com/ben-eb/caniuse-lite#license

Reply:
Example of good attribution:
https://wiki.creativecommons.org/wiki/best_practices_for_attribution

Reply:
Great analysis, I'll update all components that use caniuse-lite.